### DISCUSSION TITLE
Generate lists based on personal topic tags

### DISCUSSION THREAD

**Discussion by duskvirkus** [community/community#17283] [Other Feature Feedback, Questions, & Ideas]
**Problem:**

This perspective comes from from someone who uses stars as ways to bookmark repos and I have got so many stared projects it can often be hard to find the one I'm looking for especially when I can't think of the exact name of the repo.

As such I thought lists would be a good way to solve this problem so I began going back and using them as tags to keep track of topics I was interested in. However encountered a problem there's a limit on how many lists I can create. I also would like to be able to filter searches and stared items based on my own tags / other peoples.

**Suggestion:**

Perhaps I should give this feedback to the topics tags team however I think combing tags and lists could be useful here's what I think a solution could be:

Shift topics so they record the creator of the topic tags and make it so you can add tags on repos you don't own.
Here's an example of what that would look like in practice:
![Screenshot of modified repo info pannel that has two additional sections of tags below the existing ones. ](https://user-images.githubusercontent.com/43045568/170319993-a0dc533e-9bb6-4a6f-9eea-54153fa87250.png)

Blue = tags created by repo creator, Green = your tags tags for this repo, Fuchsia = community tags (displays top tags by everyone on this repo) (colors where just chosen for demo purposes)

Then add the ablity to have an auto generated list or search using rules that are in a dropdown that you can and/or/not together.
So some think like:
![Screenshot from 2022-05-25 11-16-10](https://user-images.githubusercontent.com/43045568/170324019-78764d69-8804-46db-8333-b24aad369ba4.png)

**Additional feedback:**

As someone who's used to filtering based on multiple labels with boolean operations, I'm often frustrated that more sites don't expose this functionality. I could be wrong but GitHub is in a rather unique position where a large percentage of people that use the site are likely have been exposed to this paradigm of querying databases. Please correct me if I'm wrong but I haven't seen this feature on github outside of possibly using the api. I'd be in favor of any feature that allows this more granular search and/or personal repo tags.

**Comment by ilekoglu** [comment]
> **Problem:**
> 
> This perspective comes from from someone who uses stars as ways to bookmark repos and I have got so many stared projects it can often be hard to find the one I'm looking for especially when I can't think of the exact name of the repo.
> 
> As such I thought lists would be a good way to solve this problem so I began going back and using them as tags to keep track of topics I was interested in. However encountered a problem there's a limit on how many lists I can create. I also would like to be able to filter searches and stared items based on my own tags / other peoples.
> 
> **Suggestion:**
> 
> Perhaps I should give this feedback to the topics tags team however I think combing tags and lists could be useful here's what I think a solution could be:
> 
> Shift topics so they record the creator of the topic tags and make it so you can add tags on repos you don't own. Here's an example of what that would look like in practice: ![Screenshot of modified repo info pannel that has two additional sections of tags below the existing ones. ](https://user-images.githubusercontent.com/43045568/170319993-a0dc533e-9bb6-4a6f-9eea-54153fa87250.png)
> 
> Blue = tags created by repo creator, Green = your tags tags for this repo, Fuchsia = community tags (displays top tags by everyone on this repo) (colors where just chosen for demo purposes)
> 
> Then add the ablity to have an auto generated list or search using rules that are in a dropdown that you can and/or/not together. So some think like: ![Screenshot from 2022-05-25 11-16-10](https://user-images.githubusercontent.com/43045568/170324019-78764d69-8804-46db-8333-b24aad369ba4.png)
> 
> **Additional feedback:**
> 
> As someone who's used to filtering based on multiple labels with boolean operations, I'm often frustrated that more sites don't expose this functionality. I could be wrong but GitHub is in a rather unique position where a large percentage of people that use the site are likely have been exposed to this paradigm of querying databases. Please correct me if I'm wrong but I haven't seen this feature on github outside of possibly using the api. I'd be in favor of any feature that allows this more granular search and/or personal repo tags.
